Source/WindowsServicify/WindowsServicify.Domain/Logger.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace WindowsServicify.Domain;
+
+public static class Logger
+{
+    public static void Log(string message)
+    {
+        string logFileName = DateTime.Now.ToString("yyyy-MM-dd") + ".log";
+        string logDirectory = @"C:\WindowsServicify\Logs";
```

Comment:
Hardcoded path, not good. The logs should be situated next to the executable that logs.

Reply:
Ups, you are right